test/hotspot/jtreg/compiler/codegen/Test8005033.java

```diff
@@ -1,4 +1,5 @@
 /*
+ * Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
```

Comment:
Since this file was first created by SAP, I would suggest adding yours after SAP's copyright line.
Thanks.

Reply:
IANAL But as simple typo fix doesn't seem worthy of a copyright claim.

Reply:
> The fix is good but I still don't understands why we don't see the test fail with:
> 
> CompileCommand: An error occurred during parsing Error: Unrecognized option 'compileony' Line: 'compileony,compiler.codegen.Test8005033::testBitCount'
> 
> ??

yes, it's a warning not an error.  It doesn't change this test, the only difference is that it was slower. 
`-Xcomp` still takes effect and compiled all methods HotSpot encounters.